include/rdma/fi_eq.h

```diff
@@ -75,10 +77,18 @@ struct fid_wait {
 	struct fi_ops_wait	*ops;
 };
 
+#ifndef _WIN32
 struct fi_mutex_cond {
 	pthread_mutex_t		*mutex;
 	pthread_cond_t		*cond;
 };
+#else /* _WIN32 */
+struct fi_critsec_cond {
+	CRITICAL_SECTION	*critsec;
+	CONDITION_VARIABLE	*cond;
+};
+#endif /* _WIN32 */
```

Comment:
Thanks for updating this.  Please add a new wait object definition.  For example, FI_WAIT_CRITSEC_COND.  And use that whenever this structure is accessed.

I'm not sure I'm being clear on this, but these wait objects are intended to be exposed as *native* wait objects used by the OS.  These are not abstractions.  Apps that want abstractions should use the OFI abstractions, like 'sread' calls or fi_wait objects.  You are still wanting to expose the windows critical section + condition variable as a "mutex + cond" abstraction.  I prefer that it be exposed as a native structure that is windows specific, similar to how fd's are exposed as native unix wait objects.